### DISCUSSION TITLE
Convert part of UI to image/pdf in .NET MAUI.

### DISCUSSION THREAD

**Discussion by Realinspirer** [dotnet/maui#8125] [General]
Is it possible to convert a part of UI or a control and its contents to a image/pdf and save it?
Or an alternate question can be - How can I print a part of UI in .NET MAUI?

**Comment by UnreachableCode** [comment]
Also looking. [IronPDF and iText7](https://ironpdf.com/blog/compare-to-other-components/itext7-csharp-html-to-pdf-alternatives/) seem like the best bet for HTML/C# but using WPF (xaml) would be incredible.

**Comment by symbiogenesis** [comment]
https://www.youtube.com/watch?v=fAIbu496unE

**Comment by mphill** [comment]
https://www.youtube.com/watch?v=fAIbu496unE talks about a new method called CaptureAsync() - that will get you an image.  Converting to a PDF will require a 3rd party library.  Outside the scope of MAUI.
